test/pkg_finish_test.rb

```diff
@@ -341,6 +341,13 @@
 
       context "if libzypp's minimalistic configuration is enabled" do
         let(:minimalistic_libzypp_config) { true }
+        let(:destdir) { tmpdir }
+
+        before do
+          conf_file = tmpdir.join(Yast::Packager::CFA::ZyppConf::PATH[1..-1])
```

Comment:
You can use `File.join` to avoid that ugly `[1..-1]`:

```suggestion
          conf_file = File.join(tmpdir, Yast::Packager::CFA::ZyppConf::PATH)
```

That looks much better IMHO...